### DISCUSSION TITLE
add or subtract

### DISCUSSION THREAD

**Discussion by dev-dhull** [filamentphp/filament#3748] [Help] [answered]
how do adding or subtracting more than two numbers in filament shown in the table column?

**Comment by danharrin** [comment, marked as answer]
`TextColumn::make('result')->getStateUsing(fn ($record) => $record->one + $record->two + $record->three)`

**Reply by farhadrahmanii** [reply]
Thank you this was very easy and very useful way sir agian thank you. 
allah bless you
